feature-libs/checkout/assets/translations/en/checkout.i18n.ts

```diff
@@ -62,6 +62,7 @@ export const checkout = {
       "Shipping addresses available to you depend on the cost center's unit",
   },
   checkoutProgress: {
+    label: 'Checkout Progress',
```

Comment:
Should be transferred

Reply:
don't worry! as explained here https://sap-cx.slack.com/archives/D02A0NHD3C0/p1643298314004500?thread_ts=1643268177.004900&cid=D02A0NHD3C0